### ISSUE TITLE
31352 Aborted?

### ISSUE THREAD

**Issue by AnonymousX1025** [pytgcalls/ntgcalls#7]
**Bug**
The processes automatically get aborted after 5 to 6 hours.

**Expected behavior**
It was expected to run without any errors. 😩

**Environment**
 - OS: Ubuntu 20.04
 - Branch: dev
 - NTgCalls Version: 1.0.0.dev23
 - Python Version: 3.10.12

**Additional context**
```
# Fatal error in: ../../../../_source/ubuntu-20.04_x86_64/webrtc/src/rtc_base/task_queue_libevent.cc, line 158

# last system error: 24
# Check failed: pipe(fds) == 0

# start: line 1: 31352 Aborted  |   (core dumped) python3 -m Modules
```

**Comment by KSKOP69** [comment]
https://github.com/pytgcalls/pytgcalls/issues/190#issuecomment-1826366902

**Comment by AnonymousX1025** [comment]
> [pytgcalls/pytgcalls#190 (comment)](https://github.com/pytgcalls/pytgcalls/issues/190#issuecomment-1826366902)

Can you explain how ?

**Comment by Ak4ssh** [comment]
@KSKOP69 https://t.me/pytgcallschat/67677
Dumb or what ?

**Comment by KSKOP69** [comment]
> @KSKOP69 https://t.me/pytgcallschat/67677 Dumb or what ?

https://github.com/pytgcalls/ntgcalls/commit/5baafd1a45ca63ed5fe7ca82644457dec3b07614

**Comment by Ak4ssh** [comment]
@KSKOP69 Then also getting error are you gonna fix it ? If no then keep quite

**Comment by KSKOP69** [comment]
> @KSKOP69 Then also getting error are you gonna fix it ? If no then keep quite

change WebRTC version maybe it will get fixed.

**Comment by Ak4ssh** [comment]
@KSKOP69 Tried anything else ?

**Comment by KSKOP69** [comment]
> @KSKOP69 Tried anything else ?

https://chromium.googlesource.com/v8/v8.git/+log/refs/branch-heads/11.9

**Comment by AnonymousX1025** [comment]
> > @KSKOP69 Tried anything else ?
> 
> https://chromium.googlesource.com/v8/v8.git/+log/refs/branch-heads/11.9

😂😂

**Comment by Laky-64** [comment]
Try with dev25

**Comment by Laky-64** [comment]
Any news? @AnonymousX1025

**Comment by AnonymousX1025** [comment]
> Try with dev25

Lemme try

**Comment by AnonymousX1025** [comment]
> Any news? @AnonymousX1025

![IMG_20231212_104709_098](https://github.com/pytgcalls/ntgcalls/assets/97216731/78f893a1-ffde-411d-992b-9eb7172ed542)

**Comment by Laky-64** [comment]
This is another bug that was fixed here d28e6813d4dda85e70ffe94ee17d7c2aaf52e081, try to use now dev27

**Comment by Laky-64** [comment]
Fixed in 1afae3f71bf5d0477f8c2170f32a1577731985f7, just use NTgCalls 1.0.1
